test_framework/terraform/aws/centos/user-data-scripts/provision_k3s_agent.sh.tpl

```diff
@@ -24,7 +24,7 @@ if [ -b "/dev/xvdh" ]; then
   mount /dev/xvdh /var/lib/longhorn
 fi
 
-until (curl -sfL https://get.k3s.io | INSTALL_K3S_EXEC="agent" K3S_URL="${k3s_server_url}" INSTALL_K3S_VERSION="${k3s_version}" K3S_CLUSTER_SECRET="${k3s_cluster_secret}" sh -); do
+until (curl -sfL https://get.k3s.io | INSTALL_K3S_EXEC="agent --token ${k3s_cluster_secret}" K3S_URL="${k3s_server_url}" INSTALL_K3S_VERSION="${k3s_version}" sh -); do
```

Comment:
Does this mean the installation way is incompatible with before? For example, using `--token` option with the agent command instead of an environment variable anymore `K3S_CLUSTER_SECRET`. Or, actually `K3S_CLUSTER_SECRET` is still working?

Reply:
![image](https://user-images.githubusercontent.com/899492/199449194-7077b6c3-d995-4fd7-904c-d09f22d9d0a4.png)

The environment var should be still valid, so I am curious why you would like to change it?

Reply:
> Does this mean the installation way is incompatible with before? 

Yes!

> Or, actually K3S_CLUSTER_SECRET is still working?

No, if using `K3S_CLUSTER_SECRET` in `v1.25.3+k3s1`, the k3s service will return error message `k3s[1724]: time="2022-11-02T09:23:46Z" level=fatal msg="cluster-secret is deprecated. Use --token instead."`, and the k3s service will not start.

For `--write-kubeconfig-mode 644`, I add this just for debugging purpose, otherwise it always needs `sudo` to execute `kubectl` on the server instances.

Reply:
Then, I think k3s doc should have better clarification :slightly_frowning_face: